filebeat/module/logstash/slowlog/_meta/fields.yml

```diff
@@ -3,22 +3,24 @@
   description: >
     slowlog
   fields:
-    - name: message
```

Comment:
Was this not populated because there is no error in CI?

Reply:
That's correct. It was never being populated. We had this conversation before 😄: https://github.com/elastic/beats/pull/10372/files#r251585088

Reply:
Perfect. Too many field discussions :-)